app/code/Magento/NewRelicReporting/Test/Mftf/Test/AdminChecksSystemConfigDependencyTest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminChecksSystemConfigDependencyTest">
+        <annotations>
+            <features value="NewRelicReporting"/>
+            <stories value="Admin is able to see the configuration fields only after enabling the feature"/>
+            <title value="Admin can see the configuration fields only after enabling the feature"/>
+            <description value="The system configs should be available only after enabling the New Relic feature."/>
+            <severity value="CRITICAL"/>
```

Comment:
I do believe we can mark this scenario with severity `MINOR`.
Even if this works as before fix it is still not braking functionality.

Reply:
Done